### PR TITLE
New design implementation

### PR DESCRIPTION
**Pull request by yotamberk**
Implement #2269

### CODE REVIEW

**Comment by yotamberk** [comment]
Capture1


Here is the work done until now. I think it's starting to look good!

**Comment by yotamberk** [comment]
Hey guys,
I've just finished fixing up the style for status bar in the bottom.
There are still all kind of tweaking up things to do still before it's perfect, but I'm afraid I'm not going to spend more of my time on it for now. I've done around 3 days of free contribution of implementing the new design and that's the max I limited myself on this project.

If you would like me to continue working on this project, we can consider my service privately .

I've enjoyed helping out and hope you like my work!

**Comment by asafMasa** [comment]
@yotamberk thanks for your hard work and efforts 🥇 
the app looks much better.

 i will discuss your suggestion with my colleagues.

**Comment by carmelorr** [comment]
Hi @yotamberk thank you very much! it looks great!!

File: src/app/app/app.component.less [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	height: 100vh;
 	display: block;
 	margin: 0;
-	font-family: 'Open Sans', sans-serif;
+	font-family: 'Lato', sans-serif;
```
Comment (asafMasa) [inline src/app/app/app.component.less]: app level fonts - main.less?

File: src/app/app/login/login/login.component.less [hunk 2]
```diff
@@ -73,7 +73,7 @@
 				}
 
 				div.try-again {
-					font-family: 'Open Sans', sans-serif;
+					font-family: 'Lato', sans-serif;
```
Comment (asafMasa) [inline src/app/app/login/login/login.component.less]: app level fonts

File: src/app/@ansyn/ansyn/less/z-index-globals.less [hunk 3]
```diff
@@ -21,7 +21,7 @@
 @z-index-menu: 4000;
 @z-index-welcome-notification: 5000;
 @z-index-overlay-overview: 5000;
-
-@z-index-spinner: 9999999;
-@z-index-spinner-before: 1000;
+@z-index-toast: 5000;
```
[the remark this answers is not in the document]
Comment (asafMasa) [reply src/app/@ansyn/ansyn/less/z-index-globals.less]: we don't use it anywhere

**Pull request merged**